Modules/Core/Common/include/itkMacro.h

```diff
@@ -441,62 +441,73 @@ OutputWindowDisplayDebugText(const char *);
-      this->SetDescription(whatmessage);                                                                               \
-    }                                                                                                                  \
-    newexcp(const std::string & file, unsigned int lineNumber)                                                         \
-      : parentexcp(file, lineNumber)                                                                                   \
-    {                                                                                                                  \
-      this->SetDescription(whatmessage);                                                                               \
-    }                                                                                                                  \
+    /* default message provides backward compatibility for a given exception type */                                   \
+    static constexpr const char * const default_exception_message = whatmessage;                                       \
+    explicit newexcp(const char * file,                                                                                \
+                     unsigned int lineNumber = 0,                                                                      \
+                     const char * desc = "None",                                                                       \
+                     const char * loc = "Unknown")                                                                     \
+      : parentexcp(std::string{ file }, lineNumber, std::string{ desc }, std::string{ loc })                           \
+    {}                                                                                                                 \
+    explicit newexcp(std::string  file,                                                                                \
+                     unsigned int lineNumber = 0,                                                                      \
+                     std::string  desc = std::string{ "None" },                                                        \
+                     std::string  loc = std::string{ "Unknown" })                                                      \
+      : parentexcp(std::move(file), lineNumber, std::move(desc), std::move(loc))                                       \
+    {}                                                                                                                 \
     itkTypeMacro(newexcp, parentexcp);                                                                                 \
   };                                                                                                                   \
   }
 
+#if defined(__INTEL_COMPILER)
+/* Explicit naming to work around Intel compiler bug.  */
```

Comment:
Is this bug still present in currently supported versions of Intel compiler? This refactoring is a good opportunity to simplify things by not treating ICC specially.

Reply:
@dzenanz Probably not, but I have no way of testing this.  I don't have access to ICC.
Removing this work-around will have a cascading effect of simplifications for many of the macros!

Do you know of anyone with the minimal intel compiler needed?

Hans

Reply:
As the [fix](https://github.com/InsightSoftwareConsortium/ITK/commit/a57cfe15840565592bc91b42ae87c43554f3a960) was done 16 years ago, it is pretty safe to assume it has been fixed by now.

Reply:
I can test ICC if it's worth doing

Reply:
@gdevenyi I have pushed a new set of changes.  Hopefully this will pass with the ICC compiler.

Thank you for testing.

Reply:
It doesn't appear to throw any issues with exception code, I'll just dump the whole stderr output here, just in case:
```
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/build/Modules/Core/Common/itkBuildInformation.cxx(62): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObject.cxx(472): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(GlobalWarningDisplay);
    ^

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkDataObject.cxx(151): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(GlobalReleaseDataFlag);
    ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 255 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "const itk::Matrix<T, NRows, NColumns>::Self &itk::Matrix<T, NRows, NColumns>::operator=(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=3U, NColumns=3U]" at line 60 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMetaDataObject.hxx"
            instantiation of "void itk::MetaDataObject<MetaDataObjectType>::SetMetaDataObjectValue(const MetaDataObjectType &) [with MetaDataObjectType=itk::Matrix<double, 3U, 3U>]" at line 46 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMetaDataObject.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkTimeStamp.cxx(56): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(GlobalTimeStamp);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMersenneTwisterRandomVariateGenerator.cxx(72): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMersenneTwisterRandomVariateGenerator.cxx(94): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMersenneTwisterRandomVariateGenerator.cxx(123): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=itk::SpacePrecisionType={double}]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=itk::SpacePrecisionType={double}]" at line 286 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Matrix(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=itk::SpacePrecisionType={double}, NRows=3U, NColumns=3U]" at line 142 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkSpatialOrientationAdapter.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(121): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(131): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(207): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(225): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(232): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkMultiThreaderBase.cxx(282): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(45): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(52): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(59): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(66): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

In file included from /home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(127):
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions_unix.cxx(138): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

In file included from /home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(127):
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions_unix.cxx(160): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

In file included from /home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions.cxx(127):
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkFloatingPointExceptions_unix.cxx(173): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(166): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(173): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(180): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(187): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(233): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(255): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(275): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(536): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(562): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(683): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(699): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkObjectFactoryBase.cxx(722): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkProgressTransformer.cxx(35): warning #177: function "itk::<unnamed>::DummyProcess::Clone" was declared but never referenced
    itkNewMacro(DummyProcess);
    ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkThreadPool.cxx(66): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkThreadPool.cxx(88): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/src/itkThreadPool.cxx(95): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(PimplGlobals);
    ^

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Video/Core/src/itkTemporalProcessObject.cxx(254): warning #68: integer conversion resulted in a change of sign
    out.SetRealDuration(ITK_INFINITE_REAL_DURATION);
                        ^

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during instantiation of class "vnl_vector_ref<T> [with T=double]" at line 72 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Mesh/src/itkSimplexMeshGeometry.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/Meta/src/itkMetaImageIO.cxx(37): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(DefaultDoublePrecision);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/Meta/src/itkMetaImageIO.cxx(1327): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(DefaultDoublePrecision);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/Meta/src/itkMetaImageIO.cxx(1334): warning #593: variable "staticGlobals" was set but never used
    itkInitGlobalsMacro(DefaultDoublePrecision);
    ^

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=itk::SpacePrecisionType={double}]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=itk::SpacePrecisionType={double}]" at line 255 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "const itk::Matrix<T, NRows, NColumns>::Self &itk::Matrix<T, NRows, NColumns>::operator=(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=2U, NColumns=2U]" at line 195 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMetaDataObject.h"
            instantiation of "bool itk::ExposeMetaData(const itk::MetaDataDictionary &, std::__cxx11::string, T &) [with T=itk::Matrix<double, 2U, 2U>]" at line 656 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/Meta/src/itkMetaImageIO.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 255 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "const itk::Matrix<T, NRows, NColumns>::Self &itk::Matrix<T, NRows, NColumns>::operator=(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=3U, NColumns=3U]" at line 1779 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/NIFTI/src/itkNiftiImageIO.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<float>() const [with T=float]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=float]" at line 175 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputVnlVectorType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformVector(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputVnlVectorType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=float, NInputDimensions=4U,
                      NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of class "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions> [with TParametersValueType=float, NInputDimensions=4U, NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::Transform(itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::NumberOfParametersType) [with TParametersValueType=float, NInputDimensions=4U, NOutputDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::MatrixOffsetTransformBase(unsigned int) [with TParametersValueType=float, NInputDimensions=4U, NOutputDimensions=4U]" at line 31 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.hxx"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=float, NDimensions=4U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::AffineTransform<float, 4U>]" at line 81 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::<unnamed>::RegisterTransformsD<TParameterType, D>::Register() [with TParameterType=float, D=4U]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=double]" at line 175 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputVnlVectorType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformVector(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputVnlVectorType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=double, NInputDimensions=4U,
                      NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of class "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions> [with TParametersValueType=double, NInputDimensions=4U, NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::Transform(itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::NumberOfParametersType) [with TParametersValueType=double, NInputDimensions=4U, NOutputDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::MatrixOffsetTransformBase(unsigned int) [with TParametersValueType=double, NInputDimensions=4U, NOutputDimensions=4U]" at line 31 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.hxx"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=double, NDimensions=4U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::AffineTransform<double, 4U>]" at line 81 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::<unnamed>::RegisterTransformsD<TParameterType, D>::Register() [with TParameterType=double, D=4U]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<float>() const [with T=float]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=float]" at line 286 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Matrix(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=float, NRows=3U, NColumns=3U]" at line 128 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.hxx"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Self itk::Matrix<T, NRows, NColumns>::operator+(const itk::Matrix<T, NRows, NColumns>::Self &) const [with T=float, NRows=3U, NColumns=3U]" at line 379 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::PreservationOfPrincipalDirectionDiffusionTensor3DReorientation(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InverseJacobianPositionType &) const
                      [with TParametersValueType=float, NInputDimensions=4U, NOutputDimensions=4U]" at line 273 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformDiffusionTensor3D(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=float,
                      NInputDimensions=4U, NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            [ 3 instantiation contexts not shown ]
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=float, NDimensions=4U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::AffineTransform<float, 4U>]" at line 81 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::<unnamed>::RegisterTransformsD<TParameterType, D>::Register() [with TParameterType=float, D=4U]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 286 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Matrix(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=3U, NColumns=3U]" at line 128 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.hxx"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Self itk::Matrix<T, NRows, NColumns>::operator+(const itk::Matrix<T, NRows, NColumns>::Self &) const [with T=double, NRows=3U, NColumns=3U]" at line 379 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::PreservationOfPrincipalDirectionDiffusionTensor3DReorientation(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InverseJacobianPositionType &) const
                      [with TParametersValueType=double, NInputDimensions=4U, NOutputDimensions=4U]" at line 273 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformDiffusionTensor3D(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=double,
                      NInputDimensions=4U, NOutputDimensions=4U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            [ 3 instantiation contexts not shown ]
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=double, NDimensions=4U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=4U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::AffineTransform<double, 4U>]" at line 81 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::<unnamed>::RegisterTransformsD<TParameterType, D>::Register() [with TParameterType=double, D=4U]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 255 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "const itk::Matrix<T, NRows, NColumns>::Self &itk::Matrix<T, NRows, NColumns>::operator=(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=3U, NColumns=3U]" at line 195 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMetaDataObject.h"
            instantiation of "bool itk::ExposeMetaData(const itk::MetaDataDictionary &, std::__cxx11::string, T &) [with T=itk::Matrix<double, 3U, 3U>]" at line 938 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/GDCM/src/itkGDCMImageIO.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 263 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "vnl_matrix_fixed<T, NColumns, NRows> itk::Matrix<T, NRows, NColumns>::GetInverse() const [with T=double, NRows=2U, NColumns=2U]" at line 162 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of "void itk::ImageBase<VImageDimension>::SetDirection(const itk::ImageBase<VImageDimension>::DirectionType &) [with VImageDimension=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of class "itk::ImageBase<VImageDimension> [with VImageDimension=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of "itk::ImageBase<VImageDimension>::ImageBase() [with VImageDimension=2U]" at line 40 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImage.hxx"
            [ 3 instantiation contexts not shown ]
            instantiation of "itk::BSplineBaseTransform<TParametersValueType, NDimensions, VSplineOrder>::BSplineBaseTransform() [with TParametersValueType=float, NDimensions=2U, VSplineOrder=3U]" at line 33 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkBSplineTransform.hxx"
            instantiation of "itk::BSplineTransform<TParametersValueType, NDimensions, VSplineOrder>::BSplineTransform() [with TParametersValueType=float, NDimensions=2U, VSplineOrder=3U]" at line 116 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkBSplineTransform.h"
            instantiation of "itk::BSplineTransform<TParametersValueType, NDimensions, VSplineOrder>::Pointer itk::BSplineTransform<TParametersValueType, NDimensions, VSplineOrder>::New() [with TParametersValueType=float, NDimensions=2U, VSplineOrder=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::BSplineTransform<float, 2U, 3U>]" at line 115 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=double]" at line 175 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputVnlVectorType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformVector(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputVnlVectorType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=double, NInputDimensions=3U,
                      NOutputDimensions=3U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of class "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions> [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::Transform(itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::NumberOfParametersType) [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::MatrixOffsetTransformBase(unsigned int) [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 31 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.hxx"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=double, NDimensions=3U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=3U]" at line 649 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.h"
            instantiation of class "itk::SpatialObject<VDimension> [with VDimension=3U]" at line 28 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.hxx"
            instantiation of "itk::EllipseSpatialObject<TDimension>::EllipseSpatialObject() [with TDimension=3U]" at line 60 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.h"
            instantiation of "itk::EllipseSpatialObject<TDimension>::Pointer itk::EllipseSpatialObject<TDimension>::New() [with TDimension=3U]" at line 48 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObjectFactory.h"
            instantiation of "void itk::SpatialObjectFactory<T>::RegisterSpatialObject() [with T=itk::EllipseSpatialObject<3U>]" at line 42 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/src/itkSpatialObjectFactoryBase.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 286 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Matrix(const itk::Matrix<T, NRows, NColumns>::Self &) [with T=double, NRows=3U, NColumns=3U]" at line 128 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.hxx"
            instantiation of "itk::Matrix<T, NRows, NColumns>::Self itk::Matrix<T, NRows, NColumns>::operator+(const itk::Matrix<T, NRows, NColumns>::Self &) const [with T=double, NRows=3U, NColumns=3U]" at line 379 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::PreservationOfPrincipalDirectionDiffusionTensor3DReorientation(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InverseJacobianPositionType &) const
                      [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 273 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputDiffusionTensor3DType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformDiffusionTensor3D(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputDiffusionTensor3DType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=double,
                      NInputDimensions=3U, NOutputDimensions=3U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            [ 4 instantiation contexts not shown ]
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=3U]" at line 649 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.h"
            instantiation of class "itk::SpatialObject<VDimension> [with VDimension=3U]" at line 28 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.hxx"
            instantiation of "itk::EllipseSpatialObject<TDimension>::EllipseSpatialObject() [with TDimension=3U]" at line 60 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.h"
            instantiation of "itk::EllipseSpatialObject<TDimension>::Pointer itk::EllipseSpatialObject<TDimension>::New() [with TDimension=3U]" at line 48 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObjectFactory.h"
            instantiation of "void itk::SpatialObjectFactory<T>::RegisterSpatialObject() [with T=itk::EllipseSpatialObject<3U>]" at line 42 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/src/itkSpatialObjectFactoryBase.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=double]" at line 120 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkPoint.hxx"
            instantiation of "vnl_vector_ref<TCoordRep> itk::Point<TCoordRep, NPointDimension>::GetVnlVector() [with TCoordRep=double, NPointDimension=2U]" at line 422 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.hxx"
            instantiation of "void itk::DisplacementFieldTransform<TParametersValueType, NDimensions>::VerifyFixedParametersInformation() [with TParametersValueType=float, NDimensions=2U]" at line 63 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.hxx"
            instantiation of class "itk::DisplacementFieldTransform<TParametersValueType, NDimensions> [with TParametersValueType=float, NDimensions=2U]" at line 63 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.hxx"
            instantiation of "itk::DisplacementFieldTransform<TParametersValueType, NDimensions>::DisplacementFieldTransform() [with TParametersValueType=float, NDimensions=2U]" at line 38 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkBSplineSmoothingOnUpdateDisplacementFieldTransform.hxx"
            instantiation of "itk::BSplineSmoothingOnUpdateDisplacementFieldTransform<TParametersValueType, NDimensions>::BSplineSmoothingOnUpdateDisplacementFieldTransform() [with TParametersValueType=float, NDimensions=2U]" at line 68 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkBSplineSmoothingOnUpdateDisplacementFieldTransform.h"
            instantiation of "itk::BSplineSmoothingOnUpdateDisplacementFieldTransform<TParametersValueType, NDimensions>::Pointer itk::BSplineSmoothingOnUpdateDisplacementFieldTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::BSplineSmoothingOnUpdateDisplacementFieldTransform<float, 2U>]" at line 161 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<double, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 2U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<double, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<double, 2U>, 2U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<double, 2U>, 2U>, itk::Image<itk::Vector<double, 2U>, 2U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions> [with TParametersValueType=double, NDimensions=2U]" at line 36 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.hxx"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::GaussianExponentialDiffeomorphicTransform() [with TParametersValueType=double, NDimensions=2U]" at line 72 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.h"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianExponentialDiffeomorphicTransform<double, 2U>]" at line 167 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<float, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 2U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<float, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 2U>, 2U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 2U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<float, 2U>, 2U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<float, 2U>, 2U>, itk::Image<itk::Vector<float, 2U>, 2U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions> [with TParametersValueType=float, NDimensions=2U]" at line 36 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.hxx"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::GaussianExponentialDiffeomorphicTransform() [with TParametersValueType=float, NDimensions=2U]" at line 72 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.h"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianExponentialDiffeomorphicTransform<float, 2U>]" at line 167 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<double, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 3U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<double, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<double, 3U>, 3U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<double, 3U>, 3U>, itk::Image<itk::Vector<double, 3U>, 3U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions> [with TParametersValueType=double, NDimensions=3U]" at line 36 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.hxx"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::GaussianExponentialDiffeomorphicTransform() [with TParametersValueType=double, NDimensions=3U]" at line 72 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.h"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianExponentialDiffeomorphicTransform<double, 3U>]" at line 168 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<float, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 3U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<float, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 3U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<float, 3U>, 3U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<float, 3U>, 3U>, itk::Image<itk::Vector<float, 3U>, 3U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions> [with TParametersValueType=float, NDimensions=3U]" at line 36 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.hxx"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::GaussianExponentialDiffeomorphicTransform() [with TParametersValueType=float, NDimensions=3U]" at line 72 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianExponentialDiffeomorphicTransform.h"
            instantiation of "itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianExponentialDiffeomorphicTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianExponentialDiffeomorphicTransform<float, 3U>]" at line 168 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<double, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 3U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<double, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<double, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<double, 2U>, 3U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<double, 2U>, 3U>, itk::Image<itk::Vector<double, 2U>, 3U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions> [with TParametersValueType=double, NDimensions=2U]" at line 41 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.hxx"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform() [with TParametersValueType=double, NDimensions=2U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.h"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<double, 2U>]" at line 171 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<float, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 3U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<float, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 2U>, 3U>, TOutputImage=itk::Image<itk::Vector<float, 2U>, 3U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<float, 2U>, 3U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<float, 2U>, 3U>, itk::Image<itk::Vector<float, 2U>, 3U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions> [with TParametersValueType=float, NDimensions=2U]" at line 41 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.hxx"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform() [with TParametersValueType=float, NDimensions=2U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.h"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=2U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<float, 2U>]" at line 171 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<double, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 4U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<double, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<double, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<double, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<double, 3U>, 4U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<double, 3U>, 4U>, itk::Image<itk::Vector<double, 3U>, 4U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions> [with TParametersValueType=double, NDimensions=3U]" at line 41 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.hxx"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform() [with TParametersValueType=double, NDimensions=3U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.h"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<double, 3U>]" at line 172 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=double]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterDoubleParameters.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<itk::Vector<float, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 4U>]" at line 214
                      of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<itk::Vector<float, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<itk::Vector<float, 3U>, 4U>, TOutputImage=itk::Image<itk::Vector<float, 3U>, 4U>]" at line 214 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx"
            instantiation of "itk::ConstNeighborhoodIterator<TImage, TBoundaryCondition>::ConstNeighborhoodIterator() [with TImage=itk::Image<itk::Vector<float, 3U>, 4U>, TBoundaryCondition=itk::ZeroFluxNeumannBoundaryCondition<itk::Image<itk::Vector<float, 3U>, 4U>, itk::Image<itk::Vector<float, 3U>, 4U>>]" at line 107 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/ImageFilterBase/include/itkVectorNeighborhoodOperatorImageFilter.hxx"
            [ 5 instantiation contexts not shown ]
            instantiation of class "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions> [with TParametersValueType=float, NDimensions=3U]" at line 41 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.hxx"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform() [with TParametersValueType=float, NDimensions=3U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Filtering/DisplacementField/include/itkGaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform.h"
            instantiation of "itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::Pointer itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=float, NDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/include/itkTransformFactory.h"
            instantiation of "void itk::TransformFactory<T>::RegisterTransform() [with T=itk::GaussianSmoothingOnUpdateTimeVaryingVelocityFieldTransform<float, 3U>]" at line 172 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx"
            instantiation of "void itk::TransformFactoryBase::RegisterTransformFactory<TParameterType>() [with TParameterType=float]" at line 29 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegisterFloatParameters.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 304 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "std::ostream &itk::operator<<(std::ostream &, const itk::Matrix<T, NRows, NColumns> &) [with T=double, NRows=4U, NColumns=4U]" at line 2095 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/MeshGifti/src/itkGiftiMeshIO.cxx"

remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 263 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "vnl_matrix_fixed<T, NColumns, NRows> itk::Matrix<T, NRows, NColumns>::GetInverse() const [with T=double, NRows=3U, NColumns=3U]" at line 432 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "const itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::InverseMatrixType &itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::GetInverseMatrix() const [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 455 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "bool itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::GetInverse(itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::Self *) const [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 278 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.hxx"
            instantiation of "bool itk::AffineTransform<TParametersValueType, NDimensions>::GetInverse(itk::AffineTransform<TParametersValueType, NDimensions>::Self *) const [with TParametersValueType=double, NDimensions=3U]" at line 568 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::SetObjectToWorldTransform(const itk::SpatialObject<VDimension>::TransformType *) [with VDimension=3U]" at line 1055 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::SetParent(itk::SpatialObject<VDimension>::Self *) [with VDimension=3U]" at line 455 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::AddChild(itk::SpatialObject<VDimension>::Self *) [with VDimension=3U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/SpatialObjects/src/itkPolygonGroupSpatialObjectXMLFile.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=double]" at line 175 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::OutputVnlVectorType itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::TransformVector(const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputVnlVectorType &, const itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::InputPointType &) const [with TParametersValueType=double, NInputDimensions=3U,
                      NOutputDimensions=3U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of class "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions> [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 43 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkTransform.hxx"
            instantiation of "itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::Transform(itk::Transform<TParametersValueType, NInputDimensions, NOutputDimensions>::NumberOfParametersType) [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx"
            instantiation of "itk::MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::MatrixOffsetTransformBase(unsigned int) [with TParametersValueType=double, NInputDimensions=3U, NOutputDimensions=3U]" at line 31 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.hxx"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::AffineTransform() [with TParametersValueType=double, NDimensions=3U]" at line 117 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkAffineTransform.h"
            instantiation of "itk::AffineTransform<TParametersValueType, NDimensions>::Pointer itk::AffineTransform<TParametersValueType, NDimensions>::New() [with TParametersValueType=double, NDimensions=3U]" at line 603 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::ComputeObjectToParentTransform() [with VDimension=3U]" at line 576 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::SetObjectToWorldTransform(const itk::SpatialObject<VDimension>::TransformType *) [with VDimension=3U]" at line 1055 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::SetParent(itk::SpatialObject<VDimension>::Self *) [with VDimension=3U]" at line 455 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/SpatialObjects/include/itkSpatialObject.hxx"
            instantiation of "void itk::SpatialObject<VDimension>::AddChild(itk::SpatialObject<VDimension>::Self *) [with VDimension=3U]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/SpatialObjects/src/itkPolygonGroupSpatialObjectXMLFile.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 263 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkMatrix.h"
            instantiation of "vnl_matrix_fixed<T, NColumns, NRows> itk::Matrix<T, NRows, NColumns>::GetInverse() const [with T=double, NRows=3U, NColumns=3U]" at line 162 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of "void itk::ImageBase<VImageDimension>::SetDirection(const itk::ImageBase<VImageDimension>::DirectionType &) [with VImageDimension=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of class "itk::ImageBase<VImageDimension> [with VImageDimension=3U]" at line 51 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageBase.hxx"
            instantiation of "itk::ImageBase<VImageDimension>::ImageBase() [with VImageDimension=3U]" at line 40 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImage.hxx"
            instantiation of "itk::Image<TPixel, VImageDimension>::Image() [with TPixel=unsigned char, VImageDimension=3U]" at line 99 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImage.h"
            instantiation of "itk::Image<TPixel, VImageDimension>::Pointer itk::Image<TPixel, VImageDimension>::New() [with TPixel=unsigned char, VImageDimension=3U]" at line 64 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageSource.hxx"
            instantiation of "itk::ProcessObject::DataObjectPointer itk::ImageSource<TOutputImage>::MakeOutput(itk::ProcessObject::DataObjectPointerArraySizeType={std::vector<itk::DataObject::Pointer, std::allocator<itk::DataObject::Pointer>>::size_type={std::size_t={unsigned long}}}) [with TOutputImage=itk::Image<unsigned char, 3U>]" at line 45 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageSource.hxx"
            instantiation of "itk::ImageSource<TOutputImage>::ImageSource() [with TOutputImage=itk::Image<unsigned char, 3U>]" at line 37 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/ImageBase/include/itkImageFileReader.hxx"
            instantiation of "itk::ImageFileReader<TOutputImage, ConvertPixelTraits>::ImageFileReader() [with TOutputImage=itk::Image<unsigned char, 3U>, ConvertPixelTraits=itk::DefaultConvertPixelTraits<unsigned char>]" at line 86 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/ImageBase/include/itkImageFileReader.h"
            instantiation of "itk::ImageFileReader<TOutputImage, ConvertPixelTraits>::Pointer itk::ImageFileReader<TOutputImage, ConvertPixelTraits>::New() [with TOutputImage=itk::Image<unsigned char, 3U>, ConvertPixelTraits=itk::DefaultConvertPixelTraits<unsigned char>]" at line 441 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_vector_ref.h(79): warning #597: "vnl_vector_ref<T>::operator vnl_vector<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_vector<T>() const { return this->as_vector(); }
             ^
          detected during:
            instantiation of class "vnl_vector_ref<T> [with T=double]" at line 135 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkPoint.hxx"
            instantiation of "vnl_vector<TCoordRep> itk::Point<TCoordRep, NPointDimension>::GetVnlVector() const [with TCoordRep=itk::SpacePrecisionType={double}, NPointDimension=6U]" at line 188 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageToImageFilter.hxx"
            instantiation of "void itk::ImageToImageFilter<TInputImage, TOutputImage>::VerifyInputInformation() const [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 46 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageToImageFilter.hxx"
            instantiation of class "itk::ImageToImageFilter<TInputImage, TOutputImage> [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 46 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkImageToImageFilter.hxx"
            instantiation of "itk::ImageToImageFilter<TInputImage, TOutputImage>::ImageToImageFilter() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 35 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ComparisonImageFilter() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.h"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::Pointer itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::New() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 518 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"
            instantiation of "int RegressionTestHelper<PixelType>(const char *, const char *, int, double, itk::SizeValueType={unsigned long}, unsigned int, bool, double, double) [with PixelType=unsigned long long]" at line 732 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 118 of
                      "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "void itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ThreadedGenerateData(const itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::OutputImageRegionType &, itk::ThreadIdType={unsigned int}) [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage> [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ComparisonImageFilter() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.h"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::Pointer itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::New() [with TInputImage=itk::Image<unsigned long long, 6U>, TOutputImage=itk::Image<unsigned long long, 6U>]" at line 518 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"
            instantiation of "int RegressionTestHelper<PixelType>(const char *, const char *, int, double, itk::SizeValueType={unsigned long}, unsigned int, bool, double, double) [with PixelType=unsigned long long]" at line 732 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 118 of
                      "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "void itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ThreadedGenerateData(const itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::OutputImageRegionType &, itk::ThreadIdType={unsigned int}) [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage> [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ComparisonImageFilter() [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.h"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::Pointer itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::New() [with TInputImage=itk::Image<long long, 6U>, TOutputImage=itk::Image<long long, 6U>]" at line 518 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"
            instantiation of "int RegressionTestHelper<PixelType>(const char *, const char *, int, double, itk::SizeValueType={unsigned long}, unsigned int, bool, double, double) [with PixelType=long long]" at line 746 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Common/include/itkZeroFluxNeumannBoundaryCondition.hxx(97): warning #177: variable "requestRegion" was declared but never referenced
    RegionType requestRegion;
               ^
          detected during:
            instantiation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::GetInputRequestedRegion(const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &, const itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::RegionType &) const [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 118 of
                      "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            implicit generation of "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage>::~ZeroFluxNeumannBoundaryCondition() [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::ZeroFluxNeumannBoundaryCondition<TInputImage, TOutputImage> [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 118 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "void itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ThreadedGenerateData(const itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::OutputImageRegionType &, itk::ThreadIdType={unsigned int}) [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of class "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage> [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 58 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::ComparisonImageFilter() [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 55 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.h"
            instantiation of "itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::Pointer itk::Testing::ComparisonImageFilter<TInputImage, TOutputImage>::New() [with TInputImage=itk::Image<double, 6U>, TOutputImage=itk::Image<double, 6U>]" at line 518 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"
            instantiation of "int RegressionTestHelper<PixelType>(const char *, const char *, int, double, itk::SizeValueType={unsigned long}, unsigned int, bool, double, double) [with PixelType=double]" at line 757 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/TestKernel/src/itkTestDriverInclude.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<double>() const [with T=double]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=double]" at line 209 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeK() [with TParametersValueType=double, NDimensions=4U]" at line 169 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeL() [with TParametersValueType=double, NDimensions=4U]" at line 151 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeWMatrix() [with TParametersValueType=double, NDimensions=4U]" at line 49 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"
            instantiation of "int itk::<unnamed>::KernelTransformHelper<TParameterType, Dimension>::InitializeWMatrix(const itk::TransformBaseTemplate<TParameterType>::Pointer &) [with TParameterType=double, Dimension=4U]" at line 158 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"
            instantiation of "void itk::TransformFileReaderTemplate<TParametersValueType>::Update() [with TParametersValueType=double]" at line 206 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"

/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/ThirdParty/VNL/src/vxl/core/vnl/vnl_matrix_ref.h(64): warning #597: "vnl_matrix_ref<T>::operator vnl_matrix<float>() const [with T=float]" will not be called for implicit or explicit conversions
    explicit operator vnl_matrix<T>() const { return vnl_matrix<T>{*this}; };
             ^
          detected during:
            instantiation of class "vnl_matrix_ref<T> [with T=float]" at line 209 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeK() [with TParametersValueType=float, NDimensions=4U]" at line 169 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeL() [with TParametersValueType=float, NDimensions=4U]" at line 151 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/Core/Transform/include/itkKernelTransform.hxx"
            instantiation of "void itk::KernelTransform<TParametersValueType, NDimensions>::ComputeWMatrix() [with TParametersValueType=float, NDimensions=4U]" at line 49 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"
            instantiation of "int itk::<unnamed>::KernelTransformHelper<TParameterType, Dimension>::InitializeWMatrix(const itk::TransformBaseTemplate<TParameterType>::Pointer &) [with TParameterType=float, Dimension=4U]" at line 158 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"
            instantiation of "void itk::TransformFileReaderTemplate<TParametersValueType>::Update() [with TParametersValueType=float]" at line 207 of "/home/m/mchakrav/gdevenyi/scratch/build/ITK/Modules/IO/TransformBase/src/itkTransformFileReader.cxx"

remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-size 
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
remark #11074: Inlining inhibited by limit max-total-size 
remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo

```